api/v1alpha1/common_types.go

```diff
@@ -1,7 +1,45 @@
 package v1alpha1
 
+import "gopkg.in/inf.v0"
+
 // NamespacedName contains the name of a object and its namespace
 type NamespacedName struct {
 	Name      string `json:"name"`
 	Namespace string `json:"namespace"`
 }
+
+// SloSpec defines what is the percentage
+type SloSpec struct {
+	// TargetAvailabilityPercentile defines the percentile number to be used
+	TargetAvailabilityPercentile string `json:"targetAvailabilityPercentile"`
```

Comment:
Sorry to bother you with the API again. Just confirmed with @RiRa12621 that availability targets are typically given in %. Also people may confuse the word percentile with percentile ranks. So, for more intuitive usage of our API, please let's use Percent.

Reply:
:(